src/main/java/essenmakanan/ingredient/IngredientList.java

```diff
@@ -1,4 +1,27 @@
 package essenmakanan.ingredient;
+import java.util.ArrayList;
 
 public class IngredientList {
+    private ArrayList<Ingredient> ingredients;
+
+    public IngredientList() {
+        ingredients = new ArrayList<>();
+    }
+
+    public void addIngredient(String ingredientName) {
+        ingredients.add(new Ingredient(ingredientName));
+    }
```

Comment:
should parameter be Ingredient object instead?

Reply:
yes, will change it